### PR TITLE
Module 2: week 2- Awesome books: with ES6

### PR DESCRIPTION
**Pull request by uno36**
In this project
- I configured linters
- added ES6 syntax to my files
- created a modules folder
- separated my code into modules

### CODE REVIEW

File: modules/UI.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    details.innerHTML = `
+          <h5>"${book.title}" by <span>${book.author}</span></h5>
+          <input type="submit" class="remove" value="Remove">        
+        `;
+    list.appendChild(details);
+    const numBooks = list.children.length;
+    if (numBooks % 2 === 0) {
+      details.style.backgroundColor = 'rgb(255, 255, 255)';
+    }
+  }
```
Comment (codepantha) [inline modules/UI.js:25]: - [ ] The add book feature is currently not working as expected. The books are neither added to the localStorage nor are they displayed on the page. Kindly make it work 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+<body>
+    <header class="header">
+        <nav class="nav-bar">
+            <h1>Awesome Books</h1>
+            <ul class="nav-links">
+                <li class="nav-item nav-item1" id="list"><a href="#" >List</span></a></li>
+                <li class="nav-item nav-item2" id="add-new"><a href="#" >Add new</a></li>
+                <li class="nav-item nav-item3" id="contact"><a href="#" >Contact</a></li>
+            </ul>
+        </nav>
```
Comment (codepantha) [inline index.html:19]: - [ ] Please display the current date below the navigation bar using the Luxon library.

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @uno36 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Used GitHub Flow
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [ ] The whole UI of your app is mangled and squished together with various elements stacked on top of each other. This has made your app practically impossible to use.

image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] I noticed that you created a whole new project for this milestone. Please in the future, checkout to a new branch, work on the requirements, and open a new PR 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (uno36) [reply index.html:19]: Please note the project is working fine on my side, I do not know what I should change 
**My UI is like this**
https://www.loom.com/share/88514fe4b1b549d5ba9a19799d458fa6
- as you can see the current  date and time are displayed
- the add book function is working as intended

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @uno36,

✔️ Implementation of date and time with luxon package

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**